docs/modules/user-guide/partials/proc_adding-container-component-to-a-devfile.adoc

```diff
@@ -74,14 +73,13 @@ components:
       image: golang
       memoryLimit: 512Mi
       mountSources: true
-      command: ['sleep', 'infinity']
 ----
 +
-The sources is mounted on a location stored in the `PROJECTS_ROOT` environment variable that is made available in the running container of the image. This location defaults to `/projects`. If `sourceMapping` is defined in the container, it overrides the 'PROJECT_ROOT' value if present and mounts the source to the path defined by `sourceMapping`.
+The sources is mounted on a location stored in the `PROJECTS_ROOT` environment variable that is made available in the running container of the image. This location defaults to `/projects`. If `sourceMapping` is defined in the container, it overrides the `PROJECT_ROOT` value and mounts the source to the path defined by `sourceMapping`.
 
 . Container Entrypoint
 +
-Use the `command` attribute of the `container` type to modify the `entrypoint` command of the container created from the image. When a devfile is used in {che-prod}, the container runs indefinitely, so the user can consistently connect to it and execute commands in it. The availability of the `sleep` command and the support for the `infinity` argument depend on the base image used in the particular images.
+Use the `command` attribute of the `container` type to modify the `entrypoint` command of the container created from the image.
```

Comment:
I simply removed reference to the `sleep` and `infinity` commands. I don't see a command attribute in the preceding source yaml, so perhaps this whole `container entrypoint` needs to be removed? Or needs more serious revisions?

Reply:
it's better to have a reference to command and args fields somewhere. it's the v1 but in v2 they are same https://github.com/devfile/api/blob/c2ce1e9dd4a1b2d29c683bb0ae96a792720f99aa/devfile.yaml#L20

Reply:
Please see my new revision.